nf_core/pipeline-template/nextflow_schema.json

```
@@ -192,7 +192,7 @@
                     "description": "Maximum amount of time that can be requested for any single job.",
                     "default": "240.h",
                     "fa_icon": "far fa-clock",
-                    "pattern": "^(\\d+(\\.\\d+)?(?:\\s*|\\.?)(s|m|h|d)\\s*)+$",
+                    "pattern": "^\\d+(\\.\\d+)?\\.?\\s*(s|m|h|day)$",
```

oops, add back something I put in before but had forgotten about:

```suggestion
                    "pattern": "^(\\d+(\\.\\d+)?\\.?\\s*(s|m|h|day)\\s*)+$",
```

---

Dropping decimals as discussed on Slack:
```suggestion
                    "pattern": "^(\\d+\\.?\\s*(s|m|h|day)\\s*)+$",
```